www/templates/control/qrc.html

```diff
@@ -0,0 +1,12 @@
+<script src="/node_modules/qrcode-generator/js/qrcode.js"></script>
+<script src="/node_modules/qrcode-generator/js/qrcode_UTF8.js"></script>
+<script src="/node_modules/angular-qrcode/angular-qrcode.js"></script>
```

Comment:
You need to have this module in `package.json`
Also, the code in `/node_modules` is typically not available to the app. Are you use this works?